### ISSUE TITLE
[Feature Request] Desktop app in Tauri

### ISSUE THREAD

**Issue by Tarek-Hasan** [silexlabs/Silex#1560]
First, thanks for your ongoing work on Silex 3.

I know it's very early to talk about a desktop app. For next version of desktop app based on Silex 3, you should consider [Tauri](https://tauri.app/). It's lighter, faster and way less complicated than Electron; which is showing its age like an old beat up car.

Hope you consider.

**Comment by lexoyo** [comment]
Very interesting thank you !
The performance and the foss parts especially
Did you use it?

**Comment by Tarek-Hasan** [comment]
I used apps built on Tauri. Though could not remember which apps as I have hundreds of apps on my pc. Also, I observed many apps built on Electron transitioned to Tauri and Edge webview (Gross, I know, right?) over the year. All because Electron devs unwilling to change their fundamental problem existed from its birth. Lack of runtime, ease of use, built-in essential features like tabs and mostly lack of an all-in-one builder. I know all of this can be done by third-party solutions, but that's not convenient.
